bl-languages/fr_FR.json

```diff
@@ -360,10 +360,9 @@
     "good-morning": "Bonjour",
     "good-afternoon": "Bonne après-midi",
     "good-evening": "Bonsoir",
-    "good-night": "Good night",
+    "good-night": "Bonne nuit",
     "hello": "Salut",
-    "there-are-no-images-for-the-page": "There are no images for the page.",
-    "select-cover-image": "Select cover image",
-    "this-plugin-depends-on-the-following-plugins": "This plugin depends on the following plugins.",
-    "no-pages-found": "No pages found"
```

Comment:
Hi Fred, what happen with the last line, you remove it.

Reply:
Oups sorry...
Fixed ;)